src/containers/search/SearchParticipants.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardSegment,
+  DatePicker,
+  Input,
+  Label,
+  PaginationToolbar,
+  SearchResults,
+  Typography,
+} from 'lattice-ui-kit';
+import {
+  DataUtils,
+  ReduxUtils,
+  useRequestState,
+} from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { SEARCH_PARTICIPANTS, clearSearchedDataAndRequestState, searchParticipants } from './actions';
+
+import { PROPERTY_TYPE_FQNS } from '../../core/edm/constants/FullyQualifiedNames';
+import { PARTICIPANT_PROFILE } from '../../core/router/Routes';
+import { goToRoute } from '../../core/router/RoutingActions';
+import { getPersonFullName } from '../../utils/PeopleUtils';
+import { SEARCH } from '../../utils/constants/ReduxStateConsts';
+
+const { DOB } = PROPERTY_TYPE_FQNS;
+const { SEARCHED_PARTICIPANTS, TOTAL_HITS } = SEARCH;
+const { isFailure, isPending, isSuccess } = ReduxUtils;
+const { getEntityKeyId, getPropertyValue } = DataUtils;
+
+const MAX_HITS = 10;
+const resultLabels = Map({
+  name: 'Name',
+  personDOB: 'Date of Birth',
+});
+
+const SearchGrid = styled.div`
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(4, 1fr);
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+
+  span:last-child {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+`;
+
+const SearchParticipants = () => {
+
+  const [formInputs, setFormInputs] = useState({ firstName: '', lastName: '' });
+  const [dob, setDOB] = useState('');
+  const [page, setPage] = useState(1);
+
+  const onInputChange = (e :SyntheticEvent<HTMLInputElement>) => {
+    const { name, value } = e.currentTarget;
+    setFormInputs({ ...formInputs, [name]: value });
+  };
+
+  const dispatch = useDispatch();
+
+  const searchCWPParticipants = (e :SyntheticEvent<HTMLInputElement> | void, startIndex :?number) => {
+    const start = startIndex || 0;
```

Comment:
you could also set the default value for `startIndex` parameter instead of declaring the `start` variable
```
const searchCWPParticipants = (e :SyntheticEvent<HTMLInputElement> | void, startIndex :?number = 0) => {
   ....
}
```

Reply:
Also since you are not using the event parameter you can remove it from the function.

Reply:
ah the reason i include the event parameter is because it is automatically passed to `onClick` function from the button. so even though i'm not using it, i cannot pass `startIndex` as the first param to this function.